hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -398,22 +398,22 @@ export class MissingContainers extends React.Component<Record<string, object>, I
           <Tabs defaultActiveKey='1'>
             <Tabs.TabPane
               key='1'
-              tab={`Missing${(missingDataSource && missingDataSource.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
+              tab={`Missing${(missingDataSource?.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
               {generateTable(missingDataSource)}
             </Tabs.TabPane>
             <Tabs.TabPane
               key='2'
-              tab={`Under-Replicated${(underReplicatedDataSource && underReplicatedDataSource.length > 0) ? ` (${underReplicatedDataSource.length})` : ''}`}>
+              tab={`Under-Replicated${(underReplicatedDataSource?.length > 0) ? ` (${underReplicatedDataSource.length})` : ''}`}>
               {generateTable(underReplicatedDataSource)}
             </Tabs.TabPane>
             <Tabs.TabPane
               key='3'
-              tab={`Over-Replicated${(overReplicatedDataSource && overReplicatedDataSource.length > 0) ? ` (${overReplicatedDataSource.length})` : ''}`}>
+              tab={`Over-Replicated${(overReplicatedDataSource?.length > 0) ? ` (${overReplicatedDataSource.length})` : ''}`}>
               {generateTable(overReplicatedDataSource)}
             </Tabs.TabPane>
             <Tabs.TabPane
               key='4'
-              tab={`Mis-Replicated${(misReplicatedDataSource && misReplicatedDataSource.length > 0) ? ` (${misReplicatedDataSource.length})` : ''}`}>
+              tab={`Mis-Replicated${(misReplicatedDataSource?.length > 0) ? ` (${misReplicatedDataSource.length})` : ''}`}>
```

Comment:
```suggestion
              tab={`Mis-Replicated${misReplicatedDataSource?.length ?? ''}`}>
```

Reply:
Done changes in latest commit  878b46ef6f3090e43a104f606bdf49ad890f0e4f